contrib/debian/bitcoin-qt.desktop

```diff
@@ -10,5 +10,5 @@ Terminal=false
 Type=Application
 Icon=bitcoin128
 MimeType=x-scheme-handler/bitcoin;
-Categories=Office;Finance;
+Categories=Office;Finance;P2P;Network;Qt;
```

Comment:
I don 't think `Qt` is a useful category. It's like adding `C++`.

Reply:
The `Qt` category is defined as "Application based on Qt libraries", which certainly applies to bitcoin-qt.

https://specifications.freedesktop.org/menu-spec/latest/apas02.html

Reply:
Huh, okay.
Well if it's part of the standard then ok...